sdk/metrics/src/jmhBasedTest/java/io/opentelemetry/sdk/metrics/internal/state/tester/ExplicitBucketHistogramTester.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.internal.state.tester;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.DoubleHistogram;
+import io.opentelemetry.sdk.metrics.Aggregation;
+import io.opentelemetry.sdk.metrics.SdkMeterProvider;
+import io.opentelemetry.sdk.metrics.internal.state.TestInstrumentType.InstrumentTester;
+import io.opentelemetry.sdk.metrics.internal.state.TestInstrumentType.TestInstrumentsState;
+import java.util.List;
+import java.util.Random;
+import org.assertj.core.util.Lists;
+
+public class ExplicitBucketHistogramTester implements InstrumentTester {
+
+  static class ExplicitHistogramState implements TestInstrumentsState {
+    DoubleHistogram doubleHistogram;
+  }
+
+  private static final int measurementsPerAttributeSet = 1_000;
+
+  @Override
+  public Aggregation testedAggregation() {
+    return Aggregation.explicitBucketHistogram(
+        Lists.newArrayList(1.0, 5.0, 10.0, 20.0, 50.0, 100.0, 500.0, 1000.0));
```

Comment:
Why these buckets instead of the defaults from `Aggregation.explicitBucketHistogram()`?

Reply:
No particular reason. 
I modified the code to use the default and take the max bucket and use it instead of the 2000 I placed.